integration/commands_freemonitoring_test.go

```diff
@@ -148,7 +148,16 @@ func TestCommandsFreeMonitoringSetFreeMonitoring(t *testing.T) {
 				actualStatus, ok := actual.Map()["state"]
 				require.True(t, ok)
 
-				assert.Equal(t, tc.expectedStatus, actualStatus)
```

Comment:
So, if `actualStatus` is `disabled`, we get status from `debug.state` and compare it. 
But we don’t do anything if `actualStatus` is not disabled - we don’t compare the results etc. Is it on purpose?

Reply:
Yes, that is true. See the response of MongoDB on the issue https://github.com/FerretDB/FerretDB/issues/2735. 

> admin> db.runCommand({setFreeMonitoring: 1, action: "enable"})
{ ok: 1 }
admin> db.runCommand({getFreeMonitoringStatus: 1})
{
  state: 'disabled',
  message: "Free monitoring is deprecated, refer to 'debug' field for actual status",
  debug: {
    state: 'enabled',
    message: 'Free monitoring has been deprecated and is not available for new MongoDB Community instances. For more information on monitoring alternatives, please see https://www.mongodb.com/docs/manual/administration/free-monitoring/',
    url: 'https://www.mongodb.com/docs/manual/administration/free-monitoring/',
    userReminder: 'Free Monitoring URL:\n' +
      'https://cloud.mongodb.com/freemonitoring/cluster/C3RNC7GDDCOCALYKJHFSGTKIU4YWAZQM'
  },
  ok: 1
}

The response contains `state: 'disabled` even though within debug `state: 'enabled'`. So just handling for `disabled` is fine.

Reply:
Ok! Thanks for the explanation.

Reply:
I misunderstood your question, I've pushed a commit to address that https://github.com/FerretDB/FerretDB/pull/2784/commits/6a5728d734a0e81b07c5c104de62c52fe84ca502

@jeremyphua letting you know as well :hugs:

Reply:
Thanks for letting me know! I was also looking through the commits and realized I might have made a mistake and the assertion should be outside of the `if` statement.